src/plugins/plugin-http.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+var shimmer = require('shimmer');
+var url = require('url');
+var isString = require('is').string;
+var merge = require('lodash.merge');
+var httpAgent = require('_http_agent');
+
+function patchModuleRoot (request, api) {
+  var labels = api.labels;
```

Comment:
You're assigning `api.labels` as `labels` here but everywhere else you're referencing it as `TraceLabels` (which is not defined) -- so maybe you meant to write TraceLabels here.